met-web/sample.env

```diff
@@ -1,8 +1,18 @@
-REACT_APP_KEYCLOAK_CLIENT=
-REACT_APP_KEYCLOAK_REALM=
-REACT_APP_KEYCLOAK_URL=
-REACT_APP_FORMIO_JWT_SECRET=
-REACT_APP_FORM_ID=
+# Keycloak auth endpoint
+REACT_APP_KEYCLOAK_URL=https://dev.loginproxy.gov.bc.ca/auth
+REACT_APP_KEYCLOAK_REALM=standard
+
+# Resource identifier for the Keycloak client
+REACT_APP_KEYCLOAK_CLIENT=modern-engagement-tools-4787
+
+# The role needed to be considered an admin
+# TODO: Allocate a dedicated role for this on SSO
+REACT_APP_KEYCLOAK_ADMIN_ROLE=create_tenant
+
+# `met-api` endpoint
 REACT_APP_API_URL=http://localhost:5000/api
+
+# `analytics-api` endpoint
 REACT_APP_ANALYTICS_API_URL=http://localhost:5001/api
-REACT_APP_ENGAGEMENT_PROJECT_TYPES=Energy-Electricity,Energy - Petroleum & Natural Gas,Food Processing,Industrial,Mines,Other,Tourist Destination Resorts,Transportation,Waste Disposal,Water Management
```

Comment:
This data seemed out of place here. I assume it's been relocated to somewhere more appropriate?

Reply:
Currently the same data is defined as the "default" [elsewhere in the repository](https://github.com/bcgov/met-public/blob/274051ecf8f024ed47f8c45b5aca70182278d291/met-web/src/config.ts#L68). I think that should ALSO be moved somewhere else, but that's out of scope for this ticket :)